10up/simple-local-avatars
# Update playground URL

### Description of the Change

This PR fixes the playground URL to use correct path for blueprints


Closes https://github.com/10up/simple-local-avatars/issues/299

### How to test the Change

- Go to https://github.com/10up/simple-local-avatars/tree/feature/update-playground
- Click on Playground Demo, confirm it isn't working

### Changelog Entry
N/A

### Credits
N/A

### Checklist:
